tidb.go

```diff
@@ -65,7 +65,15 @@ func (dm *domainMap) Get(store kv.Storage) (d *domain.Domain, err error) {
 	if !localstore.IsLocalStore(store) {
 		lease = schemaLease
 	}
-	d, err = domain.NewDomain(store, lease)
+	for i := 1; i <= defaultMaxRetries; i++ {
```

Comment:
Can we implement some function RunWithRetry(retryCount, func() {}). It should be more readable.